module1/lessons/arrays.md

```diff
@@ -1,21 +1,22 @@
 ---
-layout: lesson
+layout: page
 title: Arrays
 ---
 
 ## Learning Goals
 
-- Use Ruby syntax to declare variables that store Arrays in Ruby
+- Understand how to declare variables that store Arrays
+- Understand how to access elements within an Array
 - Use basic Array methods to manipulate data in Arrays
 
 ## Vocabulary
 
-- <span class="vocab">Array</span>
-- <span class="vocab">bracket notation</span>
-- <span class="vocab">element</span>
-- <span class="vocab">index position</span>
-- <span class="vocab">method</span>
-- <span class="vocab">square brackets `[]`</span>
+- <span class="vocab">**Array:** a Data Type that allows us to store multiple pieces of data in _one_ variable</span>
+- <span class="vocab">**bracket notation:** the syntax for accessing elements in an Array</span>
+- <span class="vocab">**element:** a piece of data in an Array</span>
+- <span class="vocab">**index position:** the location of an element in an Array</span>
+- <span class="vocab">**method:** a piece of functionality that is built into the Ruby language</span>
```

Comment:
Since methods can be declared by students and aren't always built in, I wonder if we might change this description to something like:

"a re-usable group of related instructions, sometimes already built into the Ruby (or JavaScript) language"

..or is that overkill?